deploy/astra-connector/role.yaml

```diff
@@ -32,6 +32,8 @@ rules:
     "snapshots",
     "snapshotinplacerestores",
     "snapshotrestores",
-    "astraconnectors"
+    "astraconnectors",
+    "autosupportbundleschedules",
+    "kopiavolumebackups"
```

Comment:
checking just in case; only adding `shutdownsnapshots` as a clusterrole (above), not as a role here?

Reply:
thanks for checking. That one was previously added. You might have to expand the code view to see it